application/templates/hpa.yaml

```diff
@@ -1,22 +1,24 @@
-{{- if and (.Values.autoscaling).enabled .Values.deployment }}
-{{ if .Capabilities.APIVersions.Has "autoscaling/v2/HorizontalPodAutoscaler" -}}
+{{- if and .Values.autoscaling.enabled .Values.deployment.enabled }}
+
+{{- if .Capabilities.APIVersions.Has "autoscaling/v2/HorizontalPodAutoscaler" }}
```

Comment:
Thank you for updating these check to GVK instead of GV. I'm deploying most of my apps with ArgoCD and has lot of pain previously with charts not comparing with the full GVK. See  https://github.com/argoproj/argo-cd/issues/7291

Reply:
You can thank the person behind #221 for that instead! I was originally using GV. :eyes: